app/models/dataset.rb

```diff
@@ -26,7 +24,7 @@ class Dataset < ApplicationRecord
 
   validates :frequency, inclusion: { in: %w(daily monthly quarterly annually financial-year never irregular) },
     allow_nil: true # To allow creation before setting this value
-  validates :title, presence: true, format: { with: TITLE_FORMAT }
+  validates :title, presence: true
```

Comment:
Are we not losing functionality here?

Reply:
I checked this with Jon - as long as it's present he doesn't mind. It was actually checking the string had three characters 'somewhere', which is not what we want and partly why Brakeman flagged it in the first place :-).

Reply:
We can decide if we need more specific validation on title etc later (eg character count). It sounds like it wasn't doing much useful anyway?

Reply:
Not really - it was hiding the fact we weren't doing it properly. On the other hand, it would now save a dataset with an invalid slug if you entered '[]' as the title. I guess that's unlikely.